restapi/src/main/java/io/stargate/web/docsapi/service/DocsShredder.java

```diff
@@ -73,10 +74,27 @@ private Object convertToBackendBooleanValue(boolean value, boolean numericBoolea
     if (isJson) {
       return shredJson(surfer, db, path, key, trimmed, patching);
     } else {
+      // This is a deprecated path, and is un-documented
+      // TODO: delete this after no testing relies on it
```

Comment:
can we delete now?

Reply:
YCSB tests rely on it, which is the only reason why I wanted to hold off